iml-gui/crate/src/components/stratagem/mod.rs

```diff
@@ -283,8 +283,10 @@ pub(crate) fn update(msg: Msg, cache: &ArcCache, model: &mut Model, orders: &mut
                 }
             }
             Msg::CmdSent(fetch_object) => match fetch_object.response() {
-                Ok(_) => {
-                    orders.skip();
+                Ok(x) => {
+                    let x = command_modal::Input::Commands(vec![Arc::new(x.data.command)]);
```

Comment:
This could be unified with other command_model consumers by using `fetch_json_data`, but might need  significant refactoring.

Reply:
Not worth it yet. There is a large seed 0.7 refactor looming, which redoes the fetch layer.